DevEdu.API/DevEdu.DAL/Repositories/LessonRepository.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using Dapper;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using DevEdu.DAL.Models;
+
+namespace DevEdu.DAL.Repositories
+{
+    public class LessonRepository : BaseRepository, ILessonRepository
+    {
+        private const string _lessonAddProcedure = "dbo.Lesson_Insert";
+        private const string _lessonDeleteProcedure = "dbo.Lesson_Delete";
+        private const string _lessonSelectAllProcedure = "dbo.Lesson_SelectAll";
+        private const string _lessonSelectByIdProcedure = "dbo.Lesson_SelectById";
+
+        private const string _commentAddToLessonProcedure = "dbo.Lesson_Comment_Insert";
+        private const string _commentDeleteFromLessonProcedure = "dbo.Lesson_Comment_Delete";
+
+        public LessonRepository()
+        {
+
+        }
+
+        public int AddCommentToLesson(int lessonId, int commentId)
+        {
+            return _connection.QueryFirst<int>(
+                _commentAddToLessonProcedure,
+                new
+                {
+                    lessonId,
+                    commentId
+                },
+                commandType: CommandType.StoredProcedure
+            );
+        }
+
+        public int AddLesson(LessonDto lessonDto)
+        {
+            return _connection.QueryFirst<int>(
+               _lessonAddProcedure,
+                new
+                {
+                    lessonDto.Date,
+                    lessonDto.TeacherComment,
+                    lessonDto.TeacherId
+                },
+                commandType: CommandType.StoredProcedure
+            );
+        }
+
+        public void AddTopicToLesson(int lessonId, int topicId)
+        {
+            _connection.Query("dbo.Lesson_Topic_Insert", new { lessonId, topicId });
```

Comment:
please use the same approach as methods above
or just remove this method if it's not your code and the actual version is placed in some other branch)